src/metabase/upload/types.cljc

```diff
@@ -80,13 +80,6 @@
   {::*boolean-int*  ::boolean
    ::*float-or-int* ::float})
 
-;; TODO: the set of allowed promotions should be driver-specific, because not all drivers support coercions between all
-;; types e.g. redshift does not allow coercions except between text types
-(def ^:private allowed-promotions
```

Comment:
Wow, I had totally forgotten this old method and its suggestion.

How ironic to have argued with my past self 😆 

At least my naming predilections remained the same.